rlbot_gui/story/story_challenge_setup.py

```diff
@@ -352,12 +352,16 @@ def updateStats(self, gamePacket: GameTickPacket):
                 self._last_score_by_team[team_index] = new_score
 
                 if team_index == self._human_team:
-                    last_touch_player = self._last_touch_by_team[
-                        team_index
-                    ].player_index
-                    if last_touch_player == self._human_player_index:
-                        self.stats["humanGoalsScored"] += 1
-                        print("humanGoalsScored")
+                    if self._last_touch_by_team[team_index] is not None:
```

Comment:
why not combine this with the statement above?
```
if team_index == self._human_team and self._last_touch_by_team[team_index] is not None
```

Reply:
Done